src/components/CardContainer.js

```diff
@@ -40,15 +52,46 @@ const CardContainer =  () => {
     )
   }
 
+  const selectComponents = () => {
+    if (playMode === 'play') {
+      return (
+        <div
+          style= {{
+            display: 'flex',
+            flexFlow: 'row wrap'
+          }}
+        >
+          {createCardComponents()}
+        </div>
+      ) 
+    } else if (playMode === 'won') {
+      return (
+        <div>
+          <h2>Winner!!</h2>
+        </div>
+      )
+    } else if (playMode === 'lost'){
+      return (
+        <div>
+          <h2>Loser!!</h2>
+        </div>
+      )
+    }
+  }
+
+  useEffect(() => {
+    if (playMode === 'won' || playMode === 'lost') {
+      setTimeout(() => setPlayMode('play'), 1500)
+    }
+  }, [playMode])
+
   return (
-    <div 
-        style= {{
-          display: 'flex',
-          flexFlow: 'row wrap'
-        }}>
-        {createCardComponents()}
+    <div>
+      <div>
         <h2>{score}</h2>
-      </div> 
+      </div>
+       {selectComponents()}
```

Comment:
I personally would refactor line 93 (`{selectComponents()}`) this way, so that the content returned by selectComponents is rendered directly here. I'm not 💯 to what extent it's true best practices, common practice, or just my preferred way, but I think it makes for the cleanest code when as much as possible is returned in the same place (if it's in the same component anyway), because you don't have to look around as much to understand what's going on in the component.

```jsx
      {playmode === 'play'
        ? (
          <div style={{ display: 'flex', flexFlow: 'row wrap' }}>
            {allCards.map(item => (
              <Card
                key={item}
                handleScore={handleScore}
                id={item}
              />
            ))}
          </div>
          )
        : <h2>{playMode === 'won' ? 'Winner' : 'Loser'}!!</h2>
     }
```

I almost never used ternaries when I was learning vanilla JS, but with React I used them *Schmidt voice* every day! Look for the places that are placeholders for content and use conditionals and ternaries inline (meaning, directly in the returned jsx, as opposed to calling a function that renders and returns it separately) to fill that the content. You definitely did that already with the `selectComponent` being in that placeholder section, but that code could be further reduced, like how the winner and loser code was basically doing the same thing.

LMK if any of that doesn't make sense or you have questions, and also let me know if you disagree!

Reply:
makes total sense!